src/lib/Modal.svelte

```diff
@@ -65,35 +65,27 @@
     class="mm-overlay"
     tabindex="-1"
     style={overlayStyles || undefined}
-    data-micromodal-close
```

Comment:
This seemingly big change is simply de-nesting `mm-container` out of `mm-overlay`. Now, it precedes it.